examples/Getting_Started/Getting-started-with-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,687 @@
+{
```

Comment:
<p>Maybe "After we load both the model and the workflow, we can create the ensemble graph."</p><p> </p><p>And maybe "You create the graph. The graph defines the path of the data through your full system."</p><p> </p><p>I'd use "inference operators" in plain text.  And I'd use "Triton Inference Server" when "the server" won't be clear enough.</p><p> </p><p>"In the following code block, we leverage two inference operators: the <code>TransformWorkflow</code> operator and the <code>PredictTensorflow</code> operator." (Or, plain text, "transform workflow operator and the predict TensorFlow operator" but not mixed case.  TensorFlow is a project and we treat it as a proper noun.)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>